sdk/netapp/Microsoft.Azure.Management.NetApp/src/Generated/INetAppResourceOperations.cs

```diff
@@ -98,7 +92,7 @@ public partial interface INetAppResourceOperations
         /// <exception cref="Microsoft.Rest.ValidationException">
         /// Thrown when a required parameter is null
         /// </exception>
-        Task<AzureOperationResponse<CheckAvailabilityResponse>> CheckFilePathAvailabilityWithHttpMessagesAsync(string location, string name, string type, string resourceGroup, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
+        Task<AzureOperationResponse<CheckAvailabilityResponse>> CheckFilePathAvailabilityWithHttpMessagesAsync(string location, string name, string subnetId, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
This is a breaking change that would have showed up as a swagger breaking change.  It doesn't look like this is covered by the breaking change approval.  We might be able to add a customization to support the old api, depending on the details.

Reply:
Yes, looks indeed like a braking change. It seems to have been missed. Apparently did not flag when any of the other SDK's got generated either, they have all been released btw. On first look supporting the old API may prove tricky as we are then missing the new required subnetId.  It is supported in older api versions but in this api version it is required.

This was requested by our users and the reasoning for this change in the service is that the underlying service creates subnets that are used to access ANF volumes and volume names are technically unique per subnet. This operation is intended to check if a name could be used for a new volume (if the name is available)  but the original implementation did not take the subnet into account only Account and Subscription. Meaning it could say a name is not available even if technically it should be available if the intention is to deploy the volume on a different subnet than the existing volume.

Reply:
I have added customization for BC using the last api-version. Let me know if this addresses the issue.

Reply:
Need to add a customization for this interface as well (the interface is also partial)